packages/use-contractkit/package.json

```diff
@@ -29,24 +31,31 @@
     "@celo/wallet-walletconnect-v1": "2.0.0-dev",
     "@ethersproject/providers": "^5.3.0",
     "@ledgerhq/hw-transport-webusb": "^5.43.0",
+    "@types/isomorphic-fetch": "^0.0.35",
+    "isomorphic-fetch": "^3.0.0",
     "qrcode.react": "^1.0.1",
     "react-device-detect": "^1.17.0",
     "react-loader-spinner": "^4.0.0",
     "react-modal": "^3.14.3",
     "unstated-next": "^1.1.0"
   },
   "devDependencies": {
+    "@testing-library/react-hooks": "^7.0.2",
+    "@types/jest": "^27.0.3",
     "@types/ledgerhq__hw-transport-webusb": "^4.70.1",
     "@types/node": "^16.3.0",
     "@types/qrcode.react": "^1.0.2",
     "@types/react": "^17.0.14",
     "@types/react-modal": "^3.12.1",
     "eslint-config-next": "^11.0.1",
+    "jest": "^27.4.4",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-test-renderer": "^17.0.2",
     "tailwindcss": "^2.2.4",
-    "tsc-watch": "^4.2.9",
-    "typescript": "^4.3.5",
+    "ts-jest": "^27.1.1",
+    "tsc-watch": "^4.5.0",
+    "typescript": "^4.5.3",
```

Comment:
I wonder if we could just move this dependency up to the top level package.json so all the packages are on the same version of typescript.

Reply:
will try